### PR TITLE
add mantine 

### PR DESCRIPTION
**Pull request by paigewilliams**
[asana task](https://app.asana.com/0/1200144661889303/1207340272137399/f)

adds [Mantine](https://mantine.dev/) as our chosen component library!

### CODE REVIEW

File: pages/index.tsx [hunk 1]
```diff
@@ -20,11 +24,12 @@ export default function Home() {
         <meta name="viewport" content="width=device-width, initial-scale=1" />
         <link rel="icon" href="/favicon.ico" />
       </Head>
+      <AppShell></AppShell>
       <main className={SourceSans.className}>
         <style jsx global>{`
           body {
             margin: 0;
-            color: ${theme.colors.textGray};
+            color: ${theme.colors.gray[9]};
```
Comment (paigewilliams) [inline pages/index.tsx]: @pricealexandra see usage of Mantine theme in styled-jsx

File: components/header.tsx [hunk 2]
```diff
@@ -1,32 +1,20 @@
 import Image from 'next/image';
+import { Group } from '@mantine/core';
+
 import logo from '../public/aclima-logo.svg';
-import theme from '../utils/theme';
+import classes from './header.module.css';
```
Comment (pricealexandra) [inline components/header.tsx]: seems like we do have to switch to css modules then, at least for the mantine components?

Comment (pricealexandra) [reply components/header.tsx]: where is the `header.module.css` file?

Comment (paigewilliams) [reply components/header.tsx]: gone, let me fix that line

File: components/panel.tsx [hunk 3]
```diff
@@ -1,11 +1,10 @@
-import theme from "@/utils/theme";
+import { theme } from '../theme';
 
 const Placeholder = () => {
   return (
     <div>
       <style jsx>{`
         div {
-          width: 300px;
```
Comment (pricealexandra) [inline components/panel.tsx:8]: not that it matters since it's just a placeholder, but curious about this update

File: components/panel.tsx [hunk 4]
```diff
@@ -1,11 +1,10 @@
-import theme from "@/utils/theme";
+import { theme } from '../theme';
 
 const Placeholder = () => {
   return (
     <div>
       <style jsx>{`
         div {
```
Comment (paigewilliams) [inline components/panel.tsx:7]: this PR is very much a Frankenstein approach of CSS modules, styled-jsx and mantine styles. I like using CSS modules to customize Mantine components. I am starting to think that we should use css modules for non Mantine components too?

Comment (pricealexandra) [reply components/panel.tsx:7]: sure, fine with me!

File: components/panel.tsx (hunk 3 above)
Comment (paigewilliams) [reply components/panel.tsx:8]: I removed the width since the AppShell.Navbar has a fixed width

**Comment by pricealexandra** [comment]
overall the changes look good, but I'm wondering if we can add a little left/right padding on the panel, and if there's a way for it to start higher on the page in the mobile view -- I think right now a mobile user would need to know to scroll down quite a bit to find it

**Comment by paigewilliams** [comment]
> overall the changes look good, but I'm wondering if we can add a little left/right padding on the panel, and if there's a way for it to start higher on the page in the mobile view -- I think right now a mobile user would need to know to scroll down quite a bit to find it

yes can do!

File: theme.ts [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+      '#848484',
+      '#717171',
+      '#656565',
+      '#575757',
+    ],
+  },
+  breakpoints: {
+    sm: '600px',
+  },
+  fontFamily: SourceSansFont,
```
Comment (paigewilliams) [inline theme.ts:30]: pretty cool how we can use the next/font with the mantine theme

File: .vscode/settings.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (paigewilliams) [inline .vscode/settings.json:1]: copied from https://mantine.dev/getting-started/#set-up-vs-code

**Review by pricealexandra — APPROVED** [review]
🎉

**Pull request merged**